server/pulp/server/webservices/views/utils.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+
+def generate_django_response(response, content=None, default=None, content_type='application/json'):
```

Comment:
I'd call ``response`` instead ``response_class``, and give it a default value. Also I'd make ``content`` the first argument, to accomodate the most common use case of:

```
return generate_json_response(my_data)
```